docs/references/backend/user/create-user.mdx

```diff
@@ -3,12 +3,16 @@ title: createUser()
 description: Use Clerk's Backend SDK to create a user.
 ---
 
+{/* clerk/javascript file: https://github.com/clerk/javascript/blob/main/packages/backend/src/api/endpoints/UserApi.ts#L140 */}
+
 # `createUser()`
 
 Creates a [`User`](/docs/references/backend/types/backend-user). Your user management settings determine how you should setup your user model.
 
 Any email address and phone number created using this method will be automatically marked as verified.
```

Comment:
It would be worth adding a callout with two points:
* anything `required` from Dashboard -> Users & Authentication -> Email, Phone, Username will need to be added here. 
* trying to add something here that is not enabled in Users & Authentication -> Email, Phone, Username will result in an error.